### PR TITLE
Favorites

### PR DESCRIPTION
**Pull request by bebzz06**
1- Element Author redirect to user page
2- Modal implemented in Home page
3- Local storage implemented and rendered in favorite page
4- Styling implemented

### CODE REVIEW

File: src/pages/Favorites/index.js [hunk 1]
```diff
@@ -1,4 +1,89 @@
...
+    handleCloseModal = () => {
+        this.setState({ showModal: false })
+    }
+    componentDidMount() {
+        this.getFromStorage();
+    }
+    render() {
+        const { likedPhotos, modalPhoto } = this.state;
+        //console.log(likedPhotos)
+        //console.log(modalPhoto)
```
Comment (yoloOnTheBattlefield) [inline src/pages/Favorites/index.js:45]: no commented out code

File: src/pages/Favorites/index.js [hunk 2]
```diff
@@ -1,4 +1,89 @@
...
+                            </StyledLink>
+                            <StyledCross onClick={this.handleCloseModal} />
+                        </TopWrapper>
+                        <div>{modalPhoto.description}</div>
+                        <Photo alt={modalPhoto.alt_description} src={modalPhoto.urls.regular} />
+                        <Footer >
+                            <BrokenHeartIcon><StyledBrokenHeart />{modalPhoto.likes}</BrokenHeartIcon>
+                            <StyledStar />
+                        </Footer>
+                    </ReactModal>} */}
```
Comment (yoloOnTheBattlefield) [inline src/pages/Favorites/index.js:83]: ????

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]
no commented out code

File: src/pages/Favorites/index.js (hunk 2 above)
Comment (bebzz06) [reply src/pages/Favorites/index.js:83]: I was rendering favorite as Home random pictures and forgot to get rid of it. Now is masonry. No modal implemented yet.

**Pull request merged**